source/app/powertabeditor.cpp

```diff
@@ -229,7 +229,16 @@ void PowerTabEditor::openFile(QString filename)
     if (filename.isEmpty())
         return;
 
+	int validationResult = myDocumentManager->containsDocument(filename.toStdString());
```

Comment:
Ah, I think it's tabs vs spaces - my editor is set to Insert Spaces instead of Keep Tabs.